### DISCUSSION TITLE
Do we have any examples to read edi files in spark using scala

### DISCUSSION THREAD

**Discussion by sapthagirik** [AbsaOSS/spark-data-standardization#50] [General]
could you help with any edi files to process into delta lake using spark scala example.

**Comment by Zejnilovic** [comment]
Hello @sapthagirik ,

we don't have any examples for any data formats as reading data is not the purpose of this library. That would be a job for Spark Data Source, for example, one of these https://spark.apache.org/docs/latest/sql-data-sources.html

This library is there for you when you have a DataFrame full of String columns and you want to apply a strongly typed schema on top of it. 

I had a quick look for that `EDI` file format and it seems there is nothing readily available for Spark, so I cannot even point you to the right direction.
